03_launch_mgmt_cluster.sh

```diff
@@ -319,7 +266,7 @@ function update_capm3_imports(){
 }
 
 #
-# Update the images for the CAPM3 deployment file to use local ones
+# Update the images for CAPM3 and BMO deployment file to use local ones
 #
 function update_component_image(){
   IMPORT=$1
```

Comment:
I would suggest to use COMPONENT instead and add a comment

Reply:
Sure I agree that would make sense. I just didn't want to change more things than I had to. It should be easier to see what actually changed in this way. But I can do a follow up where I change the names of things.